yesod-auth/Yesod/Auth/Message.hs

```diff
@@ -60,6 +60,7 @@ data AuthMessage =
     | ProvideIdentifier
     | SendPasswordResetEmail
     | PasswordResetPrompt
+    | CurrentPassword
```

Comment:
Ah ok, that makes sense. What do you want for the dummy value? I think I can get a few of translations so I'll toss those in at the same time.